aquality-tracking-integrations-core/src/main/java/aquality/tracking/integrations/core/Configuration.java

```diff
@@ -0,0 +1,52 @@
+package aquality.tracking.integrations.core;
+
+import lombok.Data;
+
+@Data
+public class Configuration {
+    private String host;
+    private String token;
+    private int projectId;
+    private String executor;
+
+    private boolean enabled;
+    private String suiteName;
+    private String buildName;
+    private String environment;
+    private String ciBuild;
+    private boolean debug;
+
+    public boolean isEnabled() {
+        String envVar = getEnvVar("aquality.isEnabled");
+        return envVar != null ? Boolean.parseBoolean(envVar) : enabled;
```

Comment:
can we generify this with using private generic method that takes as argument conversion Function like 'Boolean.parse' of 'String.valueOf'

Reply:
fixed